utils/soca/fig_gallery/vrfy_script.py

```diff
@@ -176,16 +176,16 @@
                                                'hs_h': [-0.1, 0.1]},
                               colormap='seismic',
                               projs=['North', 'South'],
-                              comout=os.path.join(comout, 'vrfy', 'incr')),   # sea ice increment
-                   plotConfig(grid_file=grid_file,
-                              data_file=os.path.join(comout, f'{RUN}.t'+cyc+'z.ice.incr.postproc.nc'),
-                              lats=np.arange(-60, 60, 10),
-                              variables_horiz={'aice_h': [-0.2, 0.2],
-                                               'hi_h': [-0.5, 0.5],
-                                               'hs_h': [-0.1, 0.1]},
-                              colormap='seismic',
-                              projs=['North', 'South'],
-                              comout=os.path.join(comout, 'vrfy', 'incr.postproc'))]   # sea ice increment after postprocessing
+                              comout=os.path.join(comout, 'vrfy', 'incr'))]   # sea ice increment
```

Comment:
why is this section commented out @JohnSteffen-NOAA ?

Reply:
@guillaumevernieres With the COMROOT output I have from the v1 expt, there are no *incr.postproc.nc files so the job failed.

Reply:
@JohnSteffen-NOAA  that increment is in develop now. Can you un-comment that section?